### PR TITLE
@ #2 | should support version specification for gcloud resource

### PR DESCRIPTION
**Pull request by hieptranquoc**
connect #2

### CODE REVIEW

**Comment by hieptranquoc** [comment]
@hoatle review for me, please!

File: resources/gcloud.rb [hunk 1]
```diff
@@ -1,27 +1,107 @@
 resource_name :gcloud
 
-# property :version, String, default: nil
+property :version, String, default: ''
+property :gcloud_path, String, default: '/usr/bin/gcloud'
+property :gcloud_repo, String, default: '/etc/yum.repos.d/google-cloud-sdk.repo'
```
Comment (hoatle) [inline resources/gcloud.rb]: I'm not sure  if we need "gcloud_repo" property for users to use, I don't see the use case for this.

Comment (hieptranquoc) [reply resources/gcloud.rb]: @hoatle This opinion has been fixed, please review again!

File: resources/gcloud.rb [hunk 2]
```diff
@@ -1,27 +1,107 @@
 resource_name :gcloud
 
-# property :version, String, default: nil
+property :version, String, default: ''
+property :gcloud_path, String, default: '/usr/bin/gcloud'
```
Comment (hoatle) [inline resources/gcloud.rb]: why not ":binary_path"? it's the same as helm and kubectl

Comment (hoatle) [reply resources/gcloud.rb]: and why not this default value "/usr/local/bin/gcloud"?

File: resources/gcloud.rb [hunk 3]
```diff
@@ -1,27 +1,107 @@
...
     end
 
-    apt_repository 'google-cloud-sdk' do
-      uri          'http://packages.cloud.google.com/apt'
-      distribution "cloud-sdk-#{node['lsb']['codename']}"
-      components   ['main']
-      # key 'A7317B0F'
-      # keyserver 'packages.cloud.google.com/apt/doc/apt-key.gpg'
+    execute 'link gcloud_path' do
+      command 'ln -s /opt/google-cloud-sdk/bin/gcloud /usr/bin/'
```
Comment (hoatle) [inline resources/gcloud.rb]: hard coded here?

**Comment by hoatle** [comment]
@hieptranquoc review done, please check my comments.

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**